src/app/pages/reportsdashboard/reports.service.ts

```diff
@@ -22,15 +22,15 @@ export class ReportsService implements OnDestroy {
   public dataEvents: Subject<CoreEvent> = new Subject<CoreEvent>();
   private reportsUtils: Worker;
 
-  constructor(private ws: WebSocketService, private core:CoreService) { 
+  constructor(private ws: WebSocketService, private core:CoreService) {
     //@ts-ignore
```

Comment:
Is it still necessary?

Reply:
Probably not, will remove in future PRs.